includes/acl/compression/stream/write_stats.h

```diff
@@ -161,6 +208,8 @@ namespace acl
 		writer["range_reduction"] = get_range_reduction_name(settings.range_reduction);
 		writer["has_scale"] = clip_context.has_scale;
 
+		write_decompression_times(allocator, clip, writer, allocate_context, decompress_pose, deallocate_context);
```

Comment:
I would put this under detailed since it is likely slow to compute

Reply:
And it's only going to get slower ;)